### PR TITLE
[DOC] Shadow xdg-open instead of xdg-utils-handlr usage (#53)

### PR DESCRIPTION
**Pull request by nut-3**
Documentation change that I've mentioned in #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,10 +138,14 @@ Starting with v0.10.0, commands with table output (i.e. `handlr list` and `handl
 sudo pacman -S handlr-regex
 ```
 
-Optionally you can also install `xdg-utils-handlr` from the AUR to replace `xdg-open`:
+Optionally you can also use `handlr-regex` as a replacement for `xdg-open` by putting a shadowing script into `$HOME/.local/bin` directtory (or any other `$PATH` directory). Use the following shell snippet as an example:
 
 ```sh
-paru -S xdg-utils-handlr
+echo "#!/bin/sh
```
Comment (Anomalocaridid) [inline README.md:144]: Could you change this into the script itself? I think that would be clearer and more general.

**Review by Anomalocaridid — CHANGES_REQUESTED** [review]
Thanks for the PR! Sorry for not getting to it myself. I just have a few changes to request before I merge this.

File: README.md (hunk 1 above)
Comment (nut-3) [reply README.md]: Sure, done

File: README.md [hunk 2]
```diff
@@ -138,10 +138,12 @@ Starting with v0.10.0, commands with table output (i.e. `handlr list` and `handl
...
 ```
 
-Optionally you can also install `xdg-utils-handlr` from the AUR to replace `xdg-open`:
+Optionally, you can also use `handlr-regex` as a replacement for `xdg-open` by shadowing it with a script in `$HOME/.local/bin/` (or any other user-scoped `$PATH` directory). Use the following script as an example:
 
 ```sh
-paru -S xdg-utils-handlr
+#!/bin/sh
+
+handlr open \"\$@\"
```
Comment (Anomalocaridid) [inline README.md:146]: Do the quotes still need to be escaped in this line?

Comment (nut-3) [reply README.md]: Sorry, my bad. Fixed.

**Comment by Anomalocaridid** [comment]
Alright, looks good to me. Thank you!

**Pull request merged**